<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>search</string>
    </attribute>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找内容：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>大小写敏感</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>从头开始</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>replace</string>
    </attribute>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查找内容：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>大小写敏感</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>替换为：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>全部替换（默认替换下一个）</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Search</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Search</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
